en_US/configuration/configuration.md

```diff
@@ -4009,17 +4009,34 @@ Set the type of session cluster lock. The session cluster lock is used to preven
 
 | Type | Optional Value                                                   | Default  |
 | ---- | ---------------------------------------------------------------- | -------- |
-| enum | `random`, `round_robin`, `sticky`, `hash_clientid`, `hash_topic` | `random` |
+| enum | `hash_clientid`, `hash_topic`, `local`, `random`, `round_robin`, `sticky`,  | - |
 
 #### Description
 
 Set a dispatch strategy for shared subscribers. Options are:
 
+- **hash_clientid**: Map (hash) publisher ClientID to subscriber.
+- **hash_topic**: Map (hash) source MQTT topic to subscriber.
+- **local**: Select a random subscriber in the group locally in the node which is processing the shared-publish request. If there is no such local subscriber, it fallbacks to `random` strategy 
 - **random**: Choose randomly among all subscribers.
 - **round_robin**: Enumerate subscribers in (unsorted) order.
 - **sticky**: First dispatch is random, then stick to it for all subsequent messages.
```

Comment:
maybe `The first dispatch of each session` is better?
Or will this https://github.com/emqx/emqx/issues/9062 be a bug?
The customers think the `sticky` is the same for the same `clientid` in different sessions

Reply:
> maybe `The first dispatch of each session` is better?
> Or will this [emqx/emqx#9062](https://github.com/emqx/emqx/issues/9062) be a bug?
> The customers think the `sticky` is the same for the same `clientid` in different sessions

Maybe add: “The stickiness lifecycle is bound to the publisher connection lifecycle, that is: if the publisher had to reconnect, stickiness will have to be reinitialized”

Reply:
I corrected that desc. Plz check it🤔